### PR TITLE
refactor: inline triangleCallback

### PR DESCRIPTION
**Pull request by github4jiawen**
avoid uglify error in strict mode

### CODE REVIEW

**Comment by gkjohnson** [comment]
Hi @github4jiawen -- thanks for the contribution!

Can you elaborate on what this change is for? What uglify error were you seeing and where?

**Comment by github4jiawen** [comment]
I'm using rollup with uglify plugin, during bundle it throws
> In strict mode code, functions can only be declared at top level or immediately within another function.

rollup has strict mode enabled by default, and it says
> Strictly speaking, ES modules are always in strict mode, so you shouldn't disable this without good reason.

File: src/MeshBVHNode.js [hunk 1]
```diff
@@ -287,7 +287,12 @@ MeshBVHNode.prototype.intersectsGeometry = ( function () {
 
 			if ( geometry.boundsTree ) {
 
-				function triangleCallback( tri ) {
+				arrayToBox( this.boundingData, obb2 );
+				obb2.matrix.copy( invertedMat );
+				obb2.update();
+
+				cachedMesh.geometry = geometry;
+				const res = geometry.boundsTree.shapecast( cachedMesh, box => obb2.intersectsBox( box ), function triangleCallback( tri ) {
```
Comment (gkjohnson) [inline src/MeshBVHNode.js]: Let's remove the function name `triangleCallback` because it shouldn't be needed anymore.

**Comment by gkjohnson** [comment]
Great thank you! One last change and I'll merge this!

Also can I ask what you're using this for? I'm always curious to hear about others projects.

**Comment by github4jiawen** [comment]
Done. I implemented a measurement tool for models. Thank you for this great module! It helps getting rid of the occasional hiccup.

**Comment by gkjohnson** [comment]
Thank you! If this isn't urgent I'll look into getting it published in a couple weeks or so

**Comment by github4jiawen** [comment]
Great!

**Comment by gkjohnson** [comment]
Published in v0.1.5.

Thanks again!

**Pull request merged**